client/src/components/CommentApp/selectors/index.ts

```diff
@@ -47,3 +47,9 @@ export const selectIsDirty = createSelector(
       return Array.from(comment.replies.values()).some(reply => reply.deleted || reply.originalText !== reply.text);
     });
   });
+
+export const selectCommentCount = (state: State) => (
```

Comment:
This should probably exclude resolved as well, I think?

Reply:
Fixed, thanks!